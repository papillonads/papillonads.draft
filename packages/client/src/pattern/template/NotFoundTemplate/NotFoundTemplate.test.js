import React from 'react'
import notFoundTemplateDefault from './NotFoundTemplate.render'

jest.mock('react-router-dom', () => {
  return {
    Link: () => <div />,
  }
})

describe('<NotFoundTemplate />', () => {
  describe('Snaphot', () => {
    test('must match notFoundTemplateDefault', () => {
      expect(global.renderToJSON(notFoundTemplateDefault)).toMatchSnapshot()
    })
  })
})
