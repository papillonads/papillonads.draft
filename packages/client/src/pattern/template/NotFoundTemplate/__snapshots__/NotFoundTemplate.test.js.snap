// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFoundTemplate /> Snaphot must match defaultRender 1`] = `
<div
  className=""
>
  <h2>
    some-name
  </h2>
  <div
    className=""
  >
    <h3>
      No match for 
      <code>
        some-path-name
      </code>
    </h3>
  </div>
  <div />
</div>
`;
